srcpkgs/pencil2d/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'pencil2d'
+pkgname=pencil2d
+version=0.6.2
+revision=1
+build_style=qmake
+wrksrc=pencil-${version}
+hostmakedepends="qt5-qmake"
+makedepends="qt5-xmlpatterns-devel qt5-svg-devel qt5-multimedia-devel"
+depends="qt5-multimedia qt5-svg qt5-xmlpatterns hicolor-icon-theme ffmpeg"
+short_desc="An easy, intuitive tool to make 2D hand-drawn animations"
+maintainer="ghost <gh0st@vivaldi.net>"
+license="GPL-2"
```

Comment:
'GPL-2.0-only'.
The [SPDX-Identifier](https://spdx.org/licenses/) should always be used for a license.